.github/workflows/run-acceptance-tests.yml

```diff
@@ -45,7 +45,9 @@ jobs:
     name: acceptance-test
     concurrency:
       group: acceptance-test-${{ matrix.index }} # TODO: concurrent tests across PRs can cause problems
-      cancel-in-progress: false
+      # TODO: https://github.com/pulumi/pulumi-cdk/issues/152 means that some resources in tests will have a static name
```

Comment:
The company preferred format for todos is this:

`TODO[pulumi/pulumi-cdk#152]: means that some resources..`

We do not have tooling yet to parse this out but we will one day.